include/nighthawk/common/request.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
```

Comment:
Do you think we should have more directory structure than `common`?

Reply:
Yes I agree; I opened an issue to reorganize source file layout a while ago. Just have not got around do proposing a new structure yet